Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM alpine
+FROM alpine:3.11.6
 
 ENV GOPATH /go
 
 COPY . /go/src/github.com/hound-search/hound
 
-COPY default-config.json /data/config.json
```

Comment:
Yup, it would seem as though you are right about this line. If my understanding is correct (I'm a bit of a novice with Docker): 
1. We copy `default-config.json` to `/data/config.json`
2. We then mount `/data` as a volume, ignoring whatever was there anyways
3. We start hound expecting `config.json` to be in the `/data` directory

You don't know of a clever way to make the config default to `default-config.json` if there isn't a `config.json` in the mounted volume, do you?

Reply:
If there is no `config.json` in that directory, the docker will quit with error because the entry point is like

```
ENTRYPOINT ["/go/bin/houndd", "-conf", "/data/config.json"]
```

You have to put match the name `config.json` in the `data` directory in the docker. When you run the docker, you mount the current path `$pwd` in your host system as the `data` volume for the docker. 

That's why we have to use `-v $(pwd):/data` in the docker run command.

Reply:
Right, sorry — that all makes sense. I was wondering if there was an easy way to copy `data-config.json` over in such a way that it could be used if `config.json` was missing from `/data` (i.e. in the current working directory). I don't think that's a big deal though, since no one's running Hound just to use the default config.

Reply:
Yeah. Let's avoid using the default in the docker. It is harmful and hard for users to debug.